runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingTreesOverlay.java

```diff
@@ -68,9 +71,55 @@ public Dimension render(Graphics2D graphics)
 	{
 		renderAxes(graphics);
 		renderTimers(graphics);
+		renderForestryEvents(graphics);
 		return null;
 	}
 
+	private void renderForestryEvents(Graphics2D graphics)
+	{
+		if (plugin.getSession() == null)
+		{
+			return;
+		}
+
+		if (config.highlightGreenRoots())
+		{
+			for (GameObject treeRoot : plugin.getGreenRoots())
+			{
+				if (treeRoot.getWorldLocation().distanceTo(client.getLocalPlayer().getWorldLocation()) <= 25)
+				{
+					Shape objectClickbox = treeRoot.getClickbox();
+					if (objectClickbox == null)
+					{
+						continue;
+					}
+					Color color = Color.GREEN;
+					graphics.setColor(color);
+					graphics.draw(objectClickbox);
+					graphics.setColor(ColorUtil.colorWithAlpha(color, color.getAlpha() / 5));
+					graphics.fill(objectClickbox);
+				}
+			}
+		}
+
+		if (config.highlightFlowers() && !plugin.getFlowers().isEmpty())
```

Comment:
The empty check can be removed from here, as the loop below would not execute its block if it was empty.